tests/os/freebsd/snmp/cpu-detailed.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
```

Comment:
Please do not copy paste test from another if you use snmp_standard : in the end it's the same code, we need to test it only once, and it's harder to maintain as it is duplicated.
If you want to add more test you can make a test specific for you case, for exemple checking that without parameter the zfs filesystem is correctly monitored.

Reply:
Yeah you're right, I removed tests using snmp_standard. I removed the _memory_ test too as the new machine I used (FreeBSD 14 on AWS) didn't have the needed OIDs. The previous walk was probably wrong. 
I kept the _storage_ test but I could not test the change on that machine as I can't create ZFS pools on AWS. The changes works on the client side however.